src/views/name.njk

```diff
@@ -0,0 +1,250 @@
+{% extends "layout.njk" %}
+
+{% block content %}
+
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <form class="form" action={{ props.currentUrl }} method="post">
+      <input type="hidden" name="pageType" value={{ props.pageType }}>
+
+      <h1 class="govuk-heading-xl">Limited partnership name</h1>
+
+      <div class="govuk-form-group">
+        <h1 class="govuk-label-wrapper">
+          <label class="govuk-label govuk-label--l" for="partnership_name">
+            What's the name of the limited partnership?
+          </label>
+        </h1>
+
+        <div id="partnership_name-hint" class="govuk-hint">
+          Do not enter a name ending, like Limited Partnership or LP. We'll
+          add this for you.
+        </div>
+
+        <div class="govuk-input__wrapper">
```

Comment:
We should use the Gov.uk nunjucks rather than raw HTML. They're in the prototype so we can just copy them.

Reply:
do you mean the wrapper or the input?